Graphs/Table/index.js

```diff
@@ -806,13 +815,17 @@ class Table extends AbstractGraph {
     // reset scroll data.
     resetScrollData() {
         const { disableRefresh } = this.getConfiguredProperties();
+        const {
+            isVSDService,
```

Comment:
This is a lib that should not be aware of what service this is! Please provide a way of injecting styles, similarly to how material-ui is using withStyles